src/domain/projects.rs

```diff
@@ -1,18 +1,22 @@
 use serde::{Deserialize, Serialize};
+use chrono::{Utc, DateTime};
+use chrono::serde::ts_seconds;
 use sqlx;
 
 #[derive(sqlx::FromRow, Debug, Clone, PartialEq, Serialize)]
```

Comment:
Does `derive(FromRow)` work for DateTime like this? sqlx has a feature `chrono` which adds the Chrono library with extra parsing features and I thought it was necessary to use that

Reply:
Also it would be nice to just store a timestamp without timezone in database since we convert all timestamps to UTC anyway

Reply:
Agree, also I think we should store timestamp in the same format as for sessions (both as `i64` or both as dedicated timestamp type)

Reply:
I've changed the type of timestamps to non time zone one. Also that macro which I used earlier was applicable only on timestamptz type, so i needed to add custom serializer. Also the sqlx::type::chrono:: type is now used to store timestamps.